filament/backend/include/backend/Program.h

```diff
@@ -117,6 +120,8 @@ class Program {
     Program& specializationConstants(
             utils::FixedCapacityVector<SpecializationConstant> specConstants) noexcept;
 
+    Program& pushConstants(PushConstantStruct const& pushConstants) noexcept;
```

Comment:
I would prefer a FixedCapacityVector based API, passed by value (so we can std::move), like the other APIs here.

Reply:
done